gradle.properties

```diff
@@ -1 +1,4 @@
-org.gradle.jvmargs=-Xmx2560M
+org.gradle.jvmargs=-Xmx1G -XX:SoftRefLRUPolicyMSPerMB=10 -Dlog4j2.disable.jmx=true -Dlog4j.shutdownHookEnabled=false
```

Comment:
If these do fix the issues with the AP processor it needs fixing in loom for everyone, not here.

Reply:
I don't see the point in waiting for a potential loom fix when this is affecting PR runs [right now](https://github.com/FabricMC/fabric/runs/1272351251).
In the event that a Loom fix is found, just remove this one line from the gradle.properties. There is large gain (reduced power required to build API, and less random PR run failures) for a tiny amount of technical debt.

And I did test this, and you can do it yourself--removing these parameters causes Gradle to crash when building on 1gb of ram. It does so with no (read: not consistent or large enough to be notable) slowdown with these parameters.

Reply:
I have pushed these args into loom, please let me know how it goes.

Reply:
It works fine!